tests/scripts/curves.pl

```diff
@@ -57,7 +57,7 @@ sub abort {
 
     system( "CFLAGS='-Werror -Wall -Wextra' make lib" )
         and abort "Failed to build lib: $curve\n";
-    system( "cd tests && make" ) and abort "Failed to build tests: $curve\n";
+    system( "make" ) and abort "Failed to build tests: $curve\n";
```

Comment:
In commit message "curves.pl - change test script to not depend on the implementation", I don't understand the motivation for this change. Could you please explain it in the commit message better? The commit message description should answer questions like: "Which implementation? Why don't we need to `cd tests` anymore? How does this help anything?" Thanks.